Dockerfile

```diff
@@ -38,6 +38,7 @@ COPY justfile /app/justfile
 COPY run_tests.sh /app/run_tests.sh
 COPY requirements.txt /app/requirements.txt
 COPY LICENSE /app/LICENSE
+COPY config.yaml /app/config.yaml
```

Comment:
honestly just couldnt make it work building the image locally without this COPY, perhaps I missed something?

Reply:
also wasnt using podman

Reply:
the webapp.Dockerfile uses an entrypoint script that creates this config file here https://github.com/marmotitude/object-storage-tests/blob/main/entrypoint.sh#L12

I think having a config file as part of the main image is dangerous since we publish this image publicly on ghcr. secrets should always be set at runtime

Reply:
the idea of this main Dockerfile is that we dont have to build it, Microsoft/Github builds it and host it for us, to use the object-storage-tests docker image from ghcr with your local config.yaml, you use the `just run` command that mounts the host machine's config.yaml inside the container and pass a CONFIG_PATH env var, see https://github.com/marmotitude/object-storage-tests/blob/e1d104da028fc43ecbdd83cc10dc3bbda039ad1e/justfile#L107-L116

Reply:
removed the COPY cmd from dockfile and changed logic that access prometheus_rw_url to get the one from the config.yaml mounted on runtime